components/MobileNavigation.js

```diff
@@ -37,6 +47,26 @@ export default class MobileNavigation extends Component {
     } else {
       if (this.state.showMobileNav) this.hideMobileNav()
     }
+    const result = this.findScrollSection()
+    this.setState({
+      nowSelectMenu: result
+    })
+  }
+
+  findScrollSection () {
+    const { navigationTarget } = this
+    const _docHeight = (document.height !== undefined) ? document.height : document.body.offsetHeight
+    if ((_docHeight - window.innerHeight) === window.scrollY) {
+      return navigationTarget[navigationTarget.length - 1].name
+    } else {
+      return navigationTarget.reduce((result, item) => {
+        const targetScroll = item.target.getBoundingClientRect().top + window.scrollY
+        if (window.scrollY >= (targetScroll - 50)) {
+          result = item.name
+        }
+        return result
```

Comment:
Lines 64~67 can be shortened into:

```js
        return window.scrollY >= targetScroll - 50 ? item.name : result
```

So that we can avoid re-assigning to the `result` variable.